frontend/src/features/public-form/components/FormPaymentPage/components/PaymentCard.tsx

```diff
@@ -0,0 +1,98 @@
+import { Box, Flex, Text, useDisclosure } from '@chakra-ui/react'
+
+import { ProductItem } from '~shared/types'
+import { centsToDollars, formatCurrency } from '~shared/utils/payments'
+
+import { BxsChevronDown } from '~assets/icons/BxsChevronDown'
+import Button from '~components/Button'
+import Checkbox from '~components/Checkbox'
+import Radio from '~components/Radio'
+
+import PaymentQuantityModal from './PaymentQuantityModal'
+
+const ItemQuantityButton = ({
+  product,
+  onClick,
+}: {
+  product: ProductItem
+  onClick: () => void
+}) => {
+  if (!product.data.multi_qty) {
+    return <></>
+  }
+  return (
+    <Box minWidth="6.75rem" height="2.75rem" onClick={onClick}>
+      <Button
+        rightIcon={<BxsChevronDown fontSize="1.5rem" />}
+        colorScheme="secondary"
+        aria-label="Change"
+        variant="clear"
+      >
+        Qty: {product.quantity}
+      </Button>
+    </Box>
+  )
+}
+
+export const PaymentCard = ({
+  product,
+  onItemChange,
+  isMultiSelect,
+}: {
+  product: ProductItem
+  onItemChange: (
+    productId: string,
+    isSelected: boolean,
+    selectedQuantity: number,
+  ) => void
+  isMultiSelect: boolean
+}) => {
+  const ChoiceElement = isMultiSelect ? Checkbox : Radio
+  const paymentQuantityModalDisclosure = useDisclosure()
+  return (
+    <Box w="100%">
+      <ChoiceElement
+        isChecked={product.selected}
+        onChange={() =>
+          onItemChange(product.data._id, !product.selected, product.quantity)
+        }
+        variant="fullWidth"
+        p="1rem"
+        minH="6.75rem"
+      >
```

Comment:
```suggestion
      alignItems="flex-start"
      >
```


alignment is slightly off
<img width="685" alt="Screenshot 2023-08-29 at 10 22 37 PM" src="https://github.com/opengovsg/FormSG/assets/56983748/c018e475-b1ec-4061-9628-187062819b0d">

Reply:
Fixed in dfaff6036c4ebc8bf5a14232b4a5093155c4c744